cloud-service-provider/ibm/deployable-architectures/productivity-suite/terraform.tfvars

```diff
@@ -0,0 +1,80 @@
+# Example Terraform variables file
+# Copy this file to terraform.tfvars and populate with your actual values
+
+# IBM Cloud API key - Get from IBM Cloud console
+ibmcloud_api_key = "your-ibm-cloud-api-key-here"
+
+# Path to kubeconfig for IBM IKS cluster
+kubeconfig_file = "~/.kube/ibm-iks-kubeconfig"
+
+# API Tokens - Replace with your actual tokens
+hf_token       = "hf_your-huggingface-token-here"
+openai_api_key = "your-openai-api-key-or-jwt-token-here"
+
```

Comment:
Add `opea_release` variable, default can be 0-latest (timeless) for Helm charts, or 1.3 (need to be updated for every release)